master/internal/api_experiment.go

```diff
@@ -833,6 +856,35 @@ func (a *apiServer) ActivateExperiment(
 	}
 }
 
+func (a *apiServer) ActivateExperiments(
+	ctx context.Context, req *apiv1.ActivateExperimentsRequest,
+) (*apiv1.ActivateExperimentsResponse, error) {
+	query, err := a.editableExperimentIds(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	expIDs := []int32{}
+	err = query.Where("id IN (?)", bun.In(req.ExperimentIds)).
+		Model(&expIDs).
+		Scan(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	resp := &apiv1.ActivateExperimentsResponse{ExperimentIds: []int32{}}
+	mockResp := &apiv1.ActivateExperimentResponse{}
+	for _, expID := range expIDs {
+		addr := experimentsAddr.Child(expID)
+		err = a.ask(addr, &apiv1.ActivateExperimentRequest{Id: expID}, mockResp)
+		if err == nil {
```

Comment:
Thinking about discarding these errors: when some experiment is not getting <activated>, how does the user know the reason why? how do we debug it?